tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,131 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason="https://nmstate.atlassian.net/browse/NMSTATE-230", strict=True
+)
+class TestVlanFiltering:
+    access_tag = 300
+    bridge_name = TEST_BRIDGE0
+
+    @pytest.mark.parametrize(
+        "is_native_vlan", [True, False], ids=["native", "not-native"]
+    )
+    def test_trunk_port_config(self, is_native_vlan, port0_up):
+        trunk_tags = TestVlanFiltering._generate_vlan_id_config(100, 101)
+        trunk_port_state = TestVlanFiltering._generate_vlan_filtering_config(
+            "trunk",
+            tag=self.access_tag if is_native_vlan else None,
+            trunk_tags=trunk_tags,
+            native_vlan=is_native_vlan,
+        )
+
+        port_name = port0_up[Interface.KEY][0][Interface.NAME]
+        bridge_state = _create_bridge_subtree_config((port_name,))
+        bridge_state[LinuxBridge.PORT_SUBTREE][0].update(trunk_port_state)
+        with linux_bridge(self.bridge_name, bridge_state) as desired_state:
+            assertlib.assert_state(desired_state)
+
+    @pytest.mark.parametrize(
+        "is_native_vlan", [True, False], ids=["native", "not-native"]
+    )
+    def test_multiple_trunk_ports_config(self, is_native_vlan, port0_up):
+        trunk_tags = TestVlanFiltering._generate_vlan_id_config(100, 102)
+        trunk_tags.append(
+            TestVlanFiltering._generate_vlan_id_range_config(200, 299)
+        )
+        trunk_port_state = TestVlanFiltering._generate_vlan_filtering_config(
+            "trunk",
+            tag=self.access_tag if is_native_vlan else None,
+            trunk_tags=trunk_tags,
+            native_vlan=is_native_vlan,
+        )
+
+        port_name = port0_up[Interface.KEY][0][Interface.NAME]
+        bridge_state = _create_bridge_subtree_config((port_name,))
+        bridge_state[LinuxBridge.PORT_SUBTREE][0].update(trunk_port_state)
+        with linux_bridge(self.bridge_name, bridge_state) as desired_state:
+            assertlib.assert_state(desired_state)
+
+    def test_access_port_config(self, bridge0_with_port0, port0_up):
+        access_port_state = TestVlanFiltering._generate_vlan_filtering_config(
+            "access", tag=self.access_tag
```

Comment:
`access` should probably come from the schema constants.

Reply:
Done